language/ir-testsuite/tests/misc/ignore_checks_when_exp_file_present.exp

```diff
@@ -0,0 +1,9 @@
+[0] Transaction(0)
+[1] Stage(Compiler)
+[2] Output(CompilerLog("Script(\nImports()\nDependency(\n)\nConstants(\n)\nMain(() (return ();\n\n)))"))
+[3] Output(CompiledScript(CompiledScript(CompiledScriptMut { module_handles: [], struct_handles: [], function_handles: [], function_instantiations: [], signatures: [Signature([])], identifiers: [], address_identifiers: [], constant_pool: [], type_parameters: [], parameters: SignatureIndex(0), code: CodeUnit { locals: SignatureIndex(0), code: [Ret] } })))
+[4] Stage(Verifier)
+[5] Stage(Serializer)
+[6] Stage(Runtime)
+[7] Output(TransactionOutput(TransactionOutput { write_set: WriteSet(WriteSetMut { write_set: [(AccessPath { address: a4a46d1b1421502568a4a6ac326d7250, path: 0185aa1f79e25ca70b8f54f8a0b673259fc19b1af35acb98885d99e160600387c9 }, Value(20975741f6b37b591141e96e0c60a57ce8a4a46d1b1421502568a4a6ac326d725001a4a46d1b1421502568a4a6ac326d725001a4a46d1b1421502568a4a6ac326d72500000000000000000180000000000000000a4a46d1b1421502568a4a6ac326d72500000000000000000180000000000000000a4a46d1b1421502568a4a6ac326d72500100000000000000))] }), events: [], gas_used: 16, status: Keep(EXECUTED) }))
```

Comment:
By default, I think we should just output:
```
Keep(EXECUTED)
```
I wouldn't really want any of the rest of this information as it makes it much harder to discern what is an interesting change or not.

Thoughts?

Reply:
To be clear, I understand part of this is to start small and expand over time. But I feel that we should start with the least amount of information in the exp file as possible, and add opt-ins as needed

Reply:
Should be doable. This was what I was planning to do in a subsequent PR anyways.